packages/parser/lib/Tokens.re

```diff
@@ -37,38 +9,11 @@ let show_error =
   | Eof => "Unexpected end"
   | New_line => "New line";
 
-let humanize =
-  fun
-  | EOF => "the end"
-  | IDENT(str) => "ident " ++ str
-  | BAD_IDENT => "bad  ident"
-  | FUNCTION(f) => "function " ++ f
-  | AT_KEYWORD(at) => "@ " ++ at
-  | HASH(h, _) => "hash: #" ++ h
-  | STRING(s) => {|string "|} ++ s ++ {|"|}
-  | URL(u) => "url " ++ u
-  | BAD_URL => "bad url"
-  | DELIM(d) => "delimiter " ++ d
-  | NUMBER(f) => "number: " ++ string_of_float(f)
-  | PERCENTAGE(f) =>
-    "percentage: " ++ string_of_float(f) ++ string_of_char('%')
-  | DIMENSION(f, s) => "dimension: " ++ string_of_float(f) ++ s
-  | WS => "whitespace"
-  | COLON => ":"
-  | SEMI_COLON => ";"
-  | COMMA => ","
-  | LEFT_BRACKET => "["
-  | RIGHT_BRACKET => "]"
-  | LEFT_PAREN => "("
-  | RIGHT_PAREN => ")"
-  | LEFT_BRACE => "{"
-  | RIGHT_BRACE => "}";
-
-module Parser = Css_parser;
+let string_of_char = c => String.make(1, c);
 
 let token_to_string =
```

Comment:
This is a bad idea, token_to_string is used for errors which doesn't make any sense to see `IDENT ("foo")` in the error instead of `"foo"`.

I would say to rollback this.